docs/poc2023q1/edge-scheduler.md

```diff
@@ -0,0 +1,64 @@
+### Edge scheduler
+The edge scheduler monitors the EdgePlacement, Location, and SyncTarget objects and maintains the results of matching.
+
+It starts from the kcp-scheduling-placement controller separated from kcp-dev/kcp, now being refactored towards the goal above.
+
+It works with [this snapshot](https://github.com/kcp-dev/kcp/tree/4506fdc064060b3fe82e1082533f9798b36ba7a5) of kcp.
+
+#### A short demo
+Start a kcp server with [this snapshot](https://github.com/kcp-dev/kcp/tree/4506fdc064060b3fe82e1082533f9798b36ba7a5).
+
+Point `$KUBECONFIG` to the admin kubeconfig for that kcp server.
+
+Workspace `root` is used as the edge service provider workspace.
```

Comment:
I think we should _not_ use the root workspace as the edge service provider workspace.  See how it is done in https://github.com/kcp-dev/edge-mc/blob/main/docs/poc2023q1/mailbox-controller.md#try-it

Reply:
OK will change this.

Reply:
It is changed to `root:edge`.